test/benchmark/shell_scripts/iGenVar_init.sh

```diff
@@ -0,0 +1,51 @@
+# -------- -------- Prepare logfile and runtime computation -------- -------- #
+mkdir logs
+echo "Logifle written to: initial_steps.log"
+exec 3>&1 1>>logs/initial_steps.log 2>&1                                # write stdout 1,2 to logfile 3 to console
+echo $(date)
+start=$(date +%s)                                                       # get starting date
+
+# -------- -------- get & build iGenVar -------- -------- #
+mkdir Repos
+cd Repos/
+git clone https://github.com/seqan/iGenVar.git
+wait
```

Comment:
you don't need this since the script will only continue when the previous command returns, i.e. you can remove all `wait`

you would use `wait` when you spawn a detached process (i.e. in another thread) and then wait on the PID of that process